packages/manager/modules/pci/src/components/hds/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_hds_status": "Certification",
+  "pci_projects_hds_description_part_1": "Hébergement de données de santé et certification HDS pour ce projet. ",
+  "pci_projects_hds_description_part_2_link": "Consultez notre page dédiée ",
+  "pci_projects_hds_description_part_3": " pour obtenir la liste des pays concernés.",
```

Comment:
I don't think we should split translations as we don't know how it translates in other languages. It might be hard to have the correct meaning just by joining those two chunks in German for instance

Reply:
So after reflexion i think it's better to split the complete phrase into two part for those raisons:
1 - i am able to add some angular directive (ng-click) properly into element.
2 - we avoid to add some HTML elements into trad files

But i will add a note into trad request to inform that the poth part must be take as one phrase.
Thanks @marie-j.

Reply:
@mohammed-zahaf Even though it is specified for the person who will translate, I'm not sure it will work 
Example here: 
https://github.com/ovh/manager/blame/509043218f23e6dc3b03a0b93b7cd01566212c11/packages/manager/modules/vps/src/dashboard/tile/configuration/translations/Messages_fr_FR.json#L12
https://github.com/ovh/manager/blob/509043218f23e6dc3b03a0b93b7cd01566212c11/packages/manager/modules/vps/src/dashboard/tile/configuration/translations/Messages_de_DE.json#L10
The link is not even in the same part of the sentence so if we had split the translations it would make no sense 
Best option is that the full sentence is considered as a link

Reply:
@marie-j as i discussed with @cbourgois, i used all phrase as a link.
I asked Marion, she is agree.

Thanks.